DuckDuckGo/MainViewController.swift

```diff
@@ -315,6 +315,14 @@ class MainViewController: UIViewController {
 
     }
     
+    override var supportedInterfaceOrientations: UIInterfaceOrientationMask {
+        return DefaultTutorialSettings().hasSeenOnboarding ? [.allButUpsideDown] : [.portrait]
```

Comment:
Question, will this ever not work? E.g. is it possible we'll have users that haven't seen the onboarding, but then won't be shown it?

Reply:
It's more to cover the case that you might not finish the onboarding then leave/kill the app and start it again.

Reply:
I'm a little bit weary of this solution in case we change that behaviour in the future. I could instead keep track of if onboarding is being shown, rather than checking if it has been seen. What do you think?

Reply:
This is probably going to be slightly cleaner than testing to see what view controller is visible, but I don't feel strongly either way tbh.